src/commandHandlers/iam.ts

```diff
@@ -44,7 +44,7 @@ export const command = async (arg: string, embed: MessageEmbed, message: Message
         .doc(message.author.id)
         .set({
             roles: await getUserRoles(message.member!),
-            updateAt: firebase.firestore.FieldValue.serverTimestamp()
+            updatedAt: firebase.firestore.FieldValue.serverTimestamp()
```

Comment:
I think this is fine, we just have to be careful since the DB will still have records that have the field `updateAt` (at least until we migrate the data).